common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -53,7 +65,9 @@
         <input type="hidden" class="value" name="input_${id}" id="input_${id}" value="${value|h}" />
     % endif
 
-    <span class="status ${status.classname}" id="status_${id}" aria-describedby="label_${id}"><span class="sr">${status.display_name}</span></span>
+    <%include file="status_span.html" args="status=status,
```

Comment:
In the annotation problem there are two status indicators.  One for the selected answer and one for the problem as a whole. Should this other status indicator be removed? @sstack22 @cahrens